onnx2kerastl/operation_layers.py

```diff
@@ -158,8 +156,7 @@ def handle_axis_change(target_data_format: str, config: Dict[str, Any]) -> Dict[
 
         return handle_axis_change
 
-    reduce_mean_layer.get_special_data_format_handler = get_special_data_format_handler
-    layers[node_name] = reduce_mean_layer(input_0)
+    layers[node_name] = K.mean(input_0, axis=axes, keepdims=keepdims)
```

Comment:
I think that we need this layer to create its special_data_format_handler when the layer is being converted from last to first.
How ` K.mean` is being converted from last to first now? (its axis should change accordingly)

Reply:
reverting to previous version